<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="3">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="lineEditEndereco"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButtonConnect">
          <property name="text">
           <string>Connect</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonDisconnect">
          <property name="text">
           <string>Disconnect</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="3" rowspan="8">
     <widget class="QTextBrowser" name="textBrowser"/>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Min</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QSlider" name="horizontalSliderMIN">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLCDNumber" name="lcdNumberMIN"/>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Max</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QSlider" name="horizontalSliderMAX">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLCDNumber" name="lcdNumberMAX"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Timing(s)</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QSlider" name="horizontalSliderTimings">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QLCDNumber" name="lcdNumberTimings">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="intValue" stdset="0">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButtonStart">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonStop">
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>463</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSliderMIN</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberMIN</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel">
     <x>201</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderMAX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberMAX</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>201</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderTimings</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberTimings</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>267</y>
    </hint>
    <hint type="destinationlabel">
     <x>201</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
